shortcodes.php

```diff
@@ -1027,6 +1028,40 @@ function sc_single_post($atts = Array())
 }
 add_shortcode('single_post', 'sc_single_post');
 
+/**
+ * Returns author bio markup for both
+ * single and featured posts
+ *
+ * @return string
+ * @author Cadie Brown
+ **/
+function sc_author_bio() {
+	global $post;
+
+	$author_title = get_post_meta( $post->ID, 'author_title', True );
+
+	$author_byline = get_post_meta( $post->ID, 'author_byline', True );
+	$author_byline = ( $author_byline != '' ) ? $author_byline : get_the_author();
+
+	$author_bio = get_post_meta( $post->ID, 'author_bio', True );
+
+	$featured_post_bio = is_page_template( 'featured-single-post.php' ) ? ' featured-author-bio' : '';
+
+	ob_start();
+	if ( $author_bio != '' ) :
+	?>
+		<hr>
+		<div class="author-bio-container<?php echo $featured_post_bio; ?>">
+			<p class="author-byline"><?php echo $author_byline; ?></p>
+			<p class="author-title"><?php echo $author_title; ?></p>
+			<p class="author-bio"><?php echo $author_bio; ?></p>
```

Comment:
I'd suggest using a `div` instead of a `p` for the author bio, in case the WYSIWYG field wraps the bio contents in paragraphs already.